# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295104355 28800
#      Sat Jan 15 07:12:35 2011 -0800
# Node ID 8f90ffe3c5f994c20ba603433db7d586a9a991a0
# Parent  2c5219471c861829f53a8eae7a34bf859db982e8
Add stdnormal_xxx family of functions to documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-15  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/stats.txi: Add stdnormal_xxx functions to documentation.
+
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/matrix.txi: Add undocumented postpad to documentation.
 
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/octave.texi, interpreter/poly.txi: Add undocumented 
 	polyaffine to documentation.
diff --git a/doc/interpreter/stats.txi b/doc/interpreter/stats.txi
--- a/doc/interpreter/stats.txi
+++ b/doc/interpreter/stats.txi
@@ -206,19 +206,20 @@ alphabetical order).
 &Gamma        && gampdf         && gamcdf        && gaminv&\cr
 &Geometric    && geopdf         && geocdf        && geoinv&\cr
 &Hypergeometric            && hygepdf      && hygecdf       && hygeinv&\cr
 &Kolmogorov Smirnov && {\it Not Available} && kolmogorov\_&& {\it Not Available}&\cr
 &             &&                && smirnov\_cdf &&&\cr
 &Laplace      && laplace\_pdf    && laplace\_cdf   && laplace\_inv&\cr
 &Logistic     && logistic\_pdf   && logistic\_cdf  && logistic\_inv&\cr
 &Log-Normal   && lognpdf        && logncdf       && logninv&\cr
+&Univariate Normal && normpdf   && normcdf       && norminv&\cr
 &Pascal       && nbinpdf        && nbincdf       && nbininv&\cr
-&Univariate Normal && normpdf   && normcdf       && norminv&\cr
 &Poisson      && poisspdf       && poisscdf      && poissinv&\cr
+&Standard Normal && stdnormal\_pdf  && stdnormal\_cdf && stdnormal\_inv&\cr
 &t (Student)  && tpdf           && tcdf          && tinv&\cr
 &Univariate Discrete && unidpdf && unidcdf       && unidinv&\cr
 &Uniform      && unifpdf        && unifcdf       && unifinv&\cr
 &Weibull      && wblpdf         && wblcdf        && wblinv&\cr
 \noalign{\hrule height 0.6pt}
 }}\hfill}}
 @end tex
 @ifnottex
@@ -282,28 +283,32 @@ alphabetical order).
 @item Logistic Distribution
   @tab @code{logistic_pdf}
   @tab @code{logistic_cdf}
   @tab @code{logistic_inv}
 @item Log-Normal Distribution
   @tab @code{lognpdf}
   @tab @code{logncdf}
   @tab @code{logninv}
+@item Univariate Normal Distribution
+  @tab @code{normpdf}
+  @tab @code{normcdf}
+  @tab @code{norminv}
 @item Pascal Distribution
   @tab @code{nbinpdf}
   @tab @code{nbincdf}
   @tab @code{nbininv}
-@item Univariate Normal Distribution
-  @tab @code{normpdf}
-  @tab @code{normcdf}
-  @tab @code{norminv}
 @item Poisson Distribution
   @tab @code{poisspdf}
   @tab @code{poisscdf}
   @tab @code{poissinv}
+@item Standard Normal Distribution
+  @tab @code{stdtormal_pdf}
+  @tab @code{stdtormal_cdf}
+  @tab @code{stdtormal_inv}
 @item t (Student) Distribution
   @tab @code{tpdf}
   @tab @code{tcdf}
   @tab @code{tinv}
 @item Univariate Discrete Distribution
   @tab @code{unidpdf}
   @tab @code{unidcdf}
   @tab @code{unidinv}
@@ -417,16 +422,22 @@ alphabetical order).
 @DOCSTRING(norminv)
 
 @DOCSTRING(poisspdf)
 
 @DOCSTRING(poisscdf)
 
 @DOCSTRING(poissinv)
 
+@DOCSTRING(stdnormal_pdf)
+
+@DOCSTRING(stdnormal_cdf)
+
+@DOCSTRING(stdnormal_inv)
+
 @DOCSTRING(tpdf)
 
 @DOCSTRING(tcdf)
 
 @DOCSTRING(tinv)
 
 @DOCSTRING(unidpdf)
 
@@ -597,16 +608,17 @@ The following table summarizes the avail
 & Geometric Distribution            && geornd &\cr
 & Hypergeometric Distribution       && hygernd &\cr
 & Laplace Distribution              && laplace\_rnd &\cr
 & Logistic Distribution             && logistic\_rnd &\cr
 & Log-Normal Distribution           && lognrnd &\cr
 & Pascal Distribution               && nbinrnd &\cr
 & Univariate Normal Distribution    && normrnd &\cr
 & Poisson Distribution              && poissrnd &\cr
+& Standard Normal Distribution      && stdnormal\_rnd &\cr
 & t (Student) Distribution          && trnd &\cr
 & Univariate Discrete Distribution  && unidrnd &\cr
 & Uniform Distribution              && unifrnd &\cr
 & Weibull Distribution              && wblrnd &\cr
 & Wiener Process                    && wienrnd &\cr
 \noalign{\hrule height 0.6pt}
 }}\hfill}}
 @end tex
@@ -625,16 +637,17 @@ The following table summarizes the avail
 @item Geometric Distribution            @tab @code{geornd}
 @item Hypergeometric Distribution       @tab @code{hygernd}
 @item Laplace Distribution              @tab @code{laplace_rnd}
 @item Logistic Distribution             @tab @code{logistic_rnd}
 @item Log-Normal Distribution           @tab @code{lognrnd}
 @item Pascal Distribution               @tab @code{nbinrnd}
 @item Univariate Normal Distribution    @tab @code{normrnd}
 @item Poisson Distribution              @tab @code{poissrnd}
+@item Standard Normal Distribution      @tab @code{stdnormal_rnd}
 @item t (Student) Distribution          @tab @code{trnd}
 @item Univariate Discrete Distribution  @tab @code{unidrnd}
 @item Uniform Distribution              @tab @code{unifrnd}
 @item Weibull Distribution              @tab @code{wblrnd}
 @item Wiener Process                    @tab @code{wienrnd}
 @end multitable
 @end ifnottex
 
@@ -667,16 +680,18 @@ The following table summarizes the avail
 @DOCSTRING(lognrnd)
 
 @DOCSTRING(nbinrnd)
 
 @DOCSTRING(normrnd)
 
 @DOCSTRING(poissrnd)
 
+@DOCSTRING(stdnormal_rnd)
+
 @DOCSTRING(trnd)
 
 @DOCSTRING(unidrnd)
 
 @DOCSTRING(unifrnd)
 
 @DOCSTRING(wblrnd)
 
